integration-test-groups/foundation/language/pom.xml

```diff
@@ -65,6 +57,32 @@
         </dependency>
 
         <!-- The following dependencies guarantee that this module is built after them. You can update them by running `mvn process-resources -Pformat -N` from the source tree root directory -->
+        <dependency>
+            <groupId>org.apache.camel.quarkus</groupId>
+            <artifactId>camel-quarkus-direct</artifactId>
```

Comment:
Isn't it generally *-deployment artifacts in this section ? Maybe it has changed ?

Reply:
Thanks for catching this!